pkg/state/manager.go

```diff
@@ -50,6 +43,11 @@ type Manager interface {
 	AddCA(name string, newCA util.CAType) error
 }
 
+type StateSerializer interface {
```

Comment:
Does this need to be public? It does not appear to be used elsewhere

Reply:
Nope. Does not need to be public at all.